app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileActivity.java

```diff
@@ -75,7 +77,8 @@ protected void onCreate(final Bundle savedInstanceState) {
      * @param view The {@link View} clicked
      */
     public void onSubmitClick(final View view) {
-        switch (this.validateInputFields()) {
+        final Constants.ExplicitFieldResultCodes result = this.validateInputFields();
+        switch (result) {
             case GOOD_FIELD:
```

Comment:
Can you rename `GOOD_FIELD` to something along the lines of `INPUT_CORRECT`?

Also, can you add some order to the cases in this switch-statement? Either alphabetically or based on the order of the inputs in the activity